payment-manager/src/main/java/com/qpay/paymentmanager/client/NotificationClientFeign.java

```diff
@@ -0,0 +1,16 @@
+package com.qpay.paymentmanager.client;
+
+import com.qpay.libs.models.PaymentNotification;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@FeignClient(name = "notification", url = "${webclient.services.notification-manager.base-url}")
+public interface NotificationClientFeign {
+
+    String NOTIFICATION_PATH = "/api/v1/notifications";
```

Comment:
Static, final

Reply:
intelj suggests me to not write because it's unnecessary